### PR TITLE
PI-8328: getProducts request is now sent with rounded display amounts

### PR DESCRIPTION
**Pull request by kernelkiller**
- the minimum is now rounded down to the next full amount without fractions
- the maximum is now rounded up to the next full amount without fractions
- this causes that the request is now sent with a minimum display amount is now sent with 1900 if the actual filter is set to 1950
- the commitTemporaryFilters() action got an option to reactivate the old behavior

### CODE REVIEW

File: filter/selectors/index.js [hunk 1]
```diff
@@ -143,6 +143,38 @@ export const getTemporaryFilters = state => (
   getFilters(state).temporaryFilters
 );
 
+/**
+ * Gets the temporary stored filter settings and rounds the display amounts. The minimum amount
+ * is rounded down, and the maximum amount is rounded down to the next full number.
```
Comment (Carsten89) [inline filter/selectors/index.js]: ... the maximum amount is rounded **UP** to ...

File: filter/actions/applyFilters.js [hunk 2]
```diff
@@ -12,10 +12,12 @@ import { FILTER_PATH } from '../constants';
 
 /**
  * Applies the filters to the products and navigates back to the previous route containing products.
+ * @param {boolean} roundDisplayAmounts If set to TRUE the values of the display_amount will be
```
[the remark this answers is not in the document]
Comment (kernelkiller) [reply filter/actions/applyFilters.js]: done

File: filter/actions/commitTemporaryFilters.js [hunk 3]
```diff
@@ -10,18 +10,30 @@ import { logger } from '@shopgate/pwa-core/helpers';
...
 import buildFilterParams from './helpers/buildFilterParams';
 
 /**
  * Submits the temporary state to the active filters.
+ * @param {boolean} roundDisplayAmounts If set to TRUE the values of the display_amount will be
+ *   rounded to the next full number.
  * @returns {Function} A redux thunk.
  */
-const commitTemporaryFilters = () => (dispatch, getState) => {
+const commitTemporaryFilters = (roundDisplayAmounts = true) => (dispatch, getState) => {
```
[the remark this answers is not in the document]
Comment (kernelkiller) [reply filter/actions/commitTemporaryFilters.js]: done

File: filter/selectors/index.js (hunk 1 above)
Comment (kernelkiller) [reply filter/selectors/index.js]: done

**Pull request merged**